tests/lookup/tests.py

```diff
@@ -776,6 +780,30 @@ def test_chain_date_time_lookups(self):
             ordered=False
         )
 
+    def test_comparison_against_decimal_field(self):
+        """
+        A comparison against an expression with DecimalField output_field
+        is handled correctly.  (#27498).
```

Comment:
I think the ticket reference could be omitted as the behavior is straightforward.